modules/ROOT/pages/mutations/create.adoc

```diff
@@ -1,7 +1,8 @@
 [[mutations-create]]
-= Create
+:description: This page describes how to create nodes through mutations.
+= `create`
```

Comment:
I'll only comment on this one, but I would perhaps not use backticks around these operation names.

Reply:
Any reason why? Just thinking whether other people would agree and this is a more collective opinion